sql/core/src/test/gen-java/org/apache/spark/sql/execution/datasources/parquet/test/avro/AvroArrayOfArray.java

```diff
@@ -1,39 +1,45 @@
 /**
  * Autogenerated by Avro
- * 
+ *
  * DO NOT EDIT DIRECTLY
  */
-package org.apache.spark.sql.execution.datasources.parquet.test.avro;  
+package org.apache.spark.sql.execution.datasources.parquet.test.avro;
+
+import org.apache.avro.specific.SpecificData;
+
 @SuppressWarnings("all")
 @org.apache.avro.specific.AvroGenerated
 public class AvroArrayOfArray extends org.apache.avro.specific.SpecificRecordBase implements org.apache.avro.specific.SpecificRecord {
+  private static final long serialVersionUID = -5085901832173924180L;
   public static final org.apache.avro.Schema SCHEMA$ = new org.apache.avro.Schema.Parser().parse("{\"type\":\"record\",\"name\":\"AvroArrayOfArray\",\"namespace\":\"org.apache.spark.sql.execution.datasources.parquet.test.avro\",\"fields\":[{\"name\":\"int_arrays_column\",\"type\":{\"type\":\"array\",\"items\":{\"type\":\"array\",\"items\":\"int\"}}}]}");
   public static org.apache.avro.Schema getClassSchema() { return SCHEMA$; }
   @Deprecated public java.util.List<java.util.List<java.lang.Integer>> int_arrays_column;
 
   /**
    * Default constructor.  Note that this does not initialize fields
    * to their default values from the schema.  If that is desired then
-   * one should use <code>newBuilder()</code>. 
+   * one should use <code>newBuilder()</code>.
```

Comment:
@aws-awinstan Can we remove unrelated changes? It looks hard to follow the changes.

Reply:
@HyukjinKwon Done! I had debated leaving these changes in when submitting the PR but decided to leave the newly generated files since they had some minor improvements (e.g. to the JavaDocs). Ideally these Avro files would be generated during the build process rather than checked in but that's a separate issue.